config.json

```diff
@@ -110,6 +110,12 @@
       ]
     },
     {
+      "slug": "bracket-push",
+      "difficulty": 3,
```

Comment:
It's true that once one realizes one can use a stack, it is easy. So the hard part is not writing the implementation. It is figuring out how to tackle the problem at all.

Because of this, I would actually rate *higher*, maybe 4 or 5.

If it helps to see how other tracks rated it:

`for i in tracks/*/config.json; do echo $i; jq 'if any(.exercises[]; .difficulty != 1) then .exercises | map(select(.slug == "bracket-push")) else [] end' < $i; done`

```
tracks/ceylon/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 6,
    "topics": [
      "Parsing",
      "Stacks",
      "Strings"
    ]
  }
]

tracks/csharp/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 5,
    "topics": [
      "Parsing",
      "Strings"
    ]
  }
]

tracks/elixir/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 3,
    "topics": []
  }
]

tracks/fsharp/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 7,
    "topics": [
      "Parsing",
      "Strings"
    ]
  }
]

tracks/java/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 5,
    "topics": []
  }
]

tracks/lua/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 5,
    "topics": [
      "stacks",
      "strings",
      "algorithms",
      "control-flow (if-else statements)",
      "control-flow (loops)"
    ]
  }
]

tracks/objective-c/config.json
[
  {
    "difficulty": 5,
    "slug": "bracket-push",
    "topics": [
      "Strings",
      "Conditionals",
      "Looping"
    ]
  }
]

tracks/ocaml/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 4,
    "topics": [
      "Stacks"
    ]
  }
]

tracks/rust/config.json
[
  {
    "slug": "bracket-push",
    "difficulty": 4,
    "topics": [
      "From trait",
      "stack or recursion"
    ]
  }
]

tracks/swift/config.json
[
  {
    "difficulty": 7,
    "slug": "bracket-push",
    "topics": [
      "Parsing",
      "Strings"
    ]
  }
]


```